whist/server/api/game/create.py

```diff
@@ -1,12 +1,16 @@
 """Route of /game/creation"""
 from typing import Dict, Optional
 
+import logging
+
 from fastapi import APIRouter, Depends, HTTPException
 from whist.core.user.player import Player
 
+from whist.server.services.password import PasswordService
```

Comment:
```suggestion
```

Reply:
if I take out this import, errors start to occur, so should I still take it out?

Reply:
yes you are right, I missed that line